docs/desktop-integration.rst

```diff
@@ -144,48 +144,121 @@ it will work as expected from within a sandboxed application.
 Theming
 -------
 
-Flatpak applications cannot directly use the system theme. This happens because flatpaks do not have the ability to use data files or libraries in ``/usr`` (where system themes are typically located). The solution to this was to package themes as Flatpaks, as relying upon the host to have the correct version for every flatpak defeats the portability benefits it provides. These themes are provided as `extensions <https://github.com/flatpak/flatpak/wiki/Extensions>`_, to the Freedesktop runtime when the extension point is Gtk, and to the KDE runtime when the extension point is Qt.
+Flatpak applications cannot directly use the system theme. This happens because flatpaks do not have the ability to use data files or libraries in ``/usr`` (where system themes are typically located). The solution to this is to package themes as Flatpaks, as relying upon the host to have the correct version for every flatpak defeats the portability benefits it provides. These themes are provided as :doc:`extension`, to the Freedesktop runtime for Gtk3 themes, and to the KDE runtime for Qt themes.
 
 The theming system requires Flatpak 0.8.4+ and applications using up to date ``org.gnome.Platform`` 3.24+, or ``org.freedesktop.Platform`` 1.6+, or ``org.kde.Platform`` 5.9+.
 
 Installing themes
 ^^^^^^^^^^^^^^^^^
 
-Instructions for Gtk
-````````````````````
+Instructions for Gtk themes
+```````````````````````````
 
-The current Gtk themes are packaged in the `flathub <https://flathub.org/>`_ repository which you can add (if it's not already added) by running::
+The current Gtk theme extensions are packaged in the `Flathub <https://flathub.org/>`_ repository which you can add (if it's not already added) by running::
 
   $ flatpak remote-add flathub https://flathub.org/repo/flathub.flatpakrepo
 
-To see a list of currently packaged themes you can use the command ``flatpak search gtk3theme`` (available since Flatpak version 0.10.1). In case you use an older version of Flatpak than that, you can use the command ``flatpak remote-ls flathub | grep org.gtk.Gtk3theme``. The difference in output between these two commands is that the first prints the application ID, the remote from which the theme comes and the theme's description, while the second prints only the full name of the theme's flatpak package.
+To see a list of currently packaged Gtk3 themes on Flathub, you can run::
 
-You can install themes with the command ``flatpak install flathub org.gtk.Gtk3theme.Foo``, replacing ``Foo`` with the name of the desired theme.
+ $ flatpak remote-ls --columns=application flathub | grep org.gtk.Gtk3theme
 
-Instructions for Qt
-```````````````````
+Then you can install the theme with::
 
-For the Qt theming to work, the flatpak packages kstyle and platformtheme must be installed. These are packed in the kdeapps repository which you can add by running::
+ $ flatpak install flathub org.gtk.Gtk3theme.Foo
```

Comment:
This is useless information. The only thing a user should ever do is run `flatpak update` and the theme gets auto-installed. If the theme doesn't get auto-installed then it wasn't properly configured and won't by set via the portal anyway.

Reply:
OK dropped.